src/main/java/com/company/service/CallService.java

```diff
@@ -0,0 +1,49 @@
+package com.company.service;
+
+import com.company.dao.ApplicationSettingDao;
+import com.company.dao.CallDAO;
+import com.company.model.MyCall;
+import com.company.model.Setting;
+import com.twilio.Twilio;
+import com.twilio.rest.api.v2010.account.Call;
+import org.joda.time.DateTime;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpServletRequest;
+import java.sql.Timestamp;
+
+@Service
+public class CallService {
+
+    @Autowired
+    private CallDAO callDAO;
+
+    @Autowired
+    private ApplicationSettingDao settingDao;
+
+    private final static String DIAL_CALL_SID = "DialCallSid";
+
+    public void writeCallToDB(HttpServletRequest request){
+
+        new Thread(() -> {
```

Comment:
remove. It's good idea, if you want add multi-threading, use @Async annotation. https://medium.com/@swathisprasad/multi-threading-in-spring-boot-using-completablefuture-a7ca68a0fe48

But, it's difficult in the moment, so we will work in sequential execution